buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -1,4 +1,5 @@
 object Versions {
     const val Kotlin = "1.5.21"
```

Comment:
is this in some guideline? I mean using capital letters for constants

Reply:
I thought so, although official Kotlin guidelines only suggest screaming snake case (https://kotlinlang.org/docs/coding-conventions.html#property-names). Will replace